meta-genivi-dev/recipes-sota/genivi-swm/genivi-swm_git.bb

```diff
@@ -0,0 +1,67 @@
+DESCRIPTION = "Genivi Software Loading Manager"
+SECTION = "base"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = \
+"file://${S}/LICENSE;md5=8d2127b230519ce4ad2c16070bdeb7b3"
```

Comment:
What is the "Proprietary" license here? Do we have permission to include this in an open source project?